setuptools/build_meta.py

```diff
@@ -118,7 +118,7 @@ def _file_with_extension(directory, extension):
     try:
         (file,) = matching
     except ValueError:
-        raise ValueError(
+        raise errors.InvalidConfigError(
```

Comment:
There seems to be a problem with `diffcov`. It is complaining about this line.
However, this line is tested in `test_build_meta.test_build_with_empty_setuppy`

Reply:
Revert change so `diffcov` might be happy...